pkg/storage/cacher.go

```diff
@@ -467,8 +467,8 @@ type cacheWatcher struct {
 
 func newCacheWatcher(resourceVersion uint64, initEvents []watchCacheEvent, filter FilterFunc, forget func(bool)) *cacheWatcher {
 	watcher := &cacheWatcher{
-		input:   make(chan watchCacheEvent, 10),
-		result:  make(chan watch.Event, 10),
+		input:   make(chan watchCacheEvent, 100),
```

Comment:
How do you know 10 is too small? We should have some rough calculation or expectation at least I guess.

For etcd2, the expected high throughput would be 1000/second. so 100 = 100ms. that is enough for a slow network even if all events sent to that watcher suddenly. if we use 10, then it is 10ms which is too short. 

for etcd3, we will have different values probably.